### PR TITLE
Move Origami Navigation Data into Origami Navigation Service

### PR DESCRIPTION
**Pull request by JakeChampion**
Closes https://github.com/Financial-Times/origami-navigation-data/issues/373

This places the navigation data files within the navigation service instead of them living on their own fastly infrastructure.

This will make the navigation updates faster to appear within the service because a navigation update will now cause the service to redeploy, and the new deployment will have the update immediately available.

I aimed to keep the git history for the files from the origami-navigation-data repository - I followed [this guide](https://medium.com/@ayushya/move-directory-from-one-repository-to-another-preserving-git-history-d210fa049d4b) to achieve that

### CODE REVIEW

**Comment by JakeChampion** [comment]
I'm tempted to remove the CLI which converts the yaml to json files but that should perhaps be a separate pull-request to keep this one to the minimal changes required to merge the two repositories together

File: test/integration/validation.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                                        )
+					.then(statusCode => assert.equal(statusCode, 200))
+					.catch(function() {
+						return heads(url,
+                                         	       {
+                                                	        headers: {
+                                                        	        'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36'
+	                                                        }
+        	                                        }
+                	                        ).then(statusCode => assert.equal(statusCode, 200));
```
Comment (chee) [inline test/integration/validation.test.js:59]: some very strange indentation spacing here

File: test/integration/v2/menus.test.js [hunk 2]
```diff
@@ -16,22 +17,15 @@ describe('GET /v2/menus', function () {
...
 			// Note: mock data can be found in ../mock/store.js
-			assert.deepEqual(json, {
-				menu1: {
-					label: 'Menu 1'
-				},
-				menu2: {
-					label: 'Menu 2'
-				}
-			});
+			assert.deepEqual(json, menu);
```
Comment (chee) [inline test/integration/v2/menus.test.js:20]: this is good

**Review by chee — APPROVED** [review]
this looks good to me

**Review by notlee — APPROVED** [review]
👍

**Pull request merged**